test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -948,5 +1187,5 @@ pactDeadBeef = let (TransactionHash b) = deadbeef
                in RequestKey $ Hash b
 
 -- avoiding `scientific` dep here
-getBlockHeight :: CommandResult a -> Maybe Word64
-getBlockHeight = preview (crMetaData . _Just . key "blockHeight" . _Number . to ((fromIntegral :: Integer -> Word64 ) . round . toRational))
+crGetBlockHeight :: CommandResult a -> Maybe Word64
```

Comment:
`getBlockHeight` is a bit ambiguously named. Get it from what?
The `cr` prefix makes things a bit clearer, IMO.
I made this change initially because I had defined a function for testing called `getBlockHeight`, and was surprised to see a conflict for a function doing something entirely different.